samples/basic/blink_led/src/main.c

```diff
@@ -29,18 +29,15 @@
 #elif defined(CONFIG_SOC_FAMILY_NRF)
 #define PWM_DRIVER CONFIG_PWM_NRF5_SW_0_DEV_NAME
 #define PWM_CHANNEL LED0_GPIO_PIN
-#elif defined(CONFIG_BOARD_HEXIWEAR_K64)
-#include <board.h>
-#define PWM_DRIVER	GREEN_PWM_NAME
-#define PWM_CHANNEL	GREEN_PWM_CHANNEL
 #elif defined(CONFIG_BOARD_COLIBRI_IMX7D_M4)
 #define PWM_DRIVER	PWM_1_LABEL
 #define PWM_CHANNEL	0
 #elif defined(CONFIG_SOC_FAMILY_NRF)
 #define PWM_DRIVER	CONFIG_PWM_NRF5_SW_0_DEV_NAME
 #define PWM_CHANNEL	LED0_GPIO_PIN
-#else
-#error "Choose supported PWM driver"
+#else /* get the defines from dt (based on alias 'pwm-led0') */
```

Comment:
What if PWM is not supported/not configured?
I guess for CI, we'll be able to get information from yaml/edts, but what about zephyr user that want to have a try?
Errors will be around PWM_LED0_PWM_CONTROLLER and PWM_LED0_PWM_CHANNEL not defined, which are generated so can't even grep on them... I think a new zephyr user could feel a bit lost ...

Reply:
Put the error conditions back